docs/development/Blackbox.md

```
@@ -4,81 +4,54 @@
 
-Cleanflight's `looptime` setting decides how frequently an update is saved to the flight log. The default looptime on
-Cleanflight is 3500. If you're using a looptime smaller than about 2400, you may experience some dropped frames due to
-the high required data rate. In that case you will need to reduce the sampling rate in the Blackbox settings, or
-increase your logger's baudrate to 250000. See the later section on configuring the Blackbox feature for details.
+Betaflight's `looptime` setting decides how frequently an update is saved to the flight log. The default looptime on Betaflight is 3500. If you're using a looptime smaller than about 2400, you may experience some dropped frames due to the high required data rate. In that case you will need to reduce the sampling rate in the Blackbox settings, or increase your logger's baudrate to 250000. See the later section on configuring the Blackbox feature for details.
 
 ## Setting up logging
 
-First, you must enable the Blackbox feature. In the [Cleanflight Configurator][] enter the Configuration tab,
-tick the "BLACKBOX" feature at the bottom of the page, and click "Save and reboot"
+First, you must enable the Blackbox feature. In the [Betaflight App][] enter the Configuration tab, tick the "BLACKBOX" feature at the bottom of the page, and click "Save and reboot"
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "betaflight app"

(MD052, reference-links-images)